recipe/meta.yaml

```diff
@@ -1,28 +1,31 @@
-{% set version = "0.1.6" %}
+{% set version = "0.2.3" %}
 
 package:
   name: clickhouse-sqlalchemy
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/c/clickhouse-sqlalchemy/clickhouse-sqlalchemy-{{ version }}.tar.gz
-  sha256: 402d112e2c7e8c7f28fd2691c5efaf701c1c19831c10a38c9355a07dd8d308a9
+  sha256: 6df4c125db428b856f8a7ec392190d9a4a2bb281eecb4fca2202e9d49ee4d55f
 
 build:
   number: 0
-  noarch: python
-  script: pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  # clickhouse-driver isn't available on s390x and aarch64.
+  skip: True  # [py<37 or (linux and (s390x or aarch64))]
+  script: pip install . --no-deps --no-build-isolation -vv
```

Comment:
I thought we prefer '{PYTHON} -m pip install . '

Reply:
Fixed